### PR TITLE
docs: fix typos and missing documentation

### PR DESCRIPTION
**Pull request by MathieuPuech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +44,9 @@ The accessibility column indicates whether the functionality is accessible in it
 | [radio](./packages/radio)                       | [![radio](https://img.shields.io/npm/v/@lion/radio.svg)](https://www.npmjs.com/package/@lion/radio)                                  | Radio from element                                     | ✔️                         |
```
Comment (jorenbroekema) [inline README.md:44]: I guess we can remove @lion/radio too since it's deprecated like checkbox

Comment (MathieuPuech) [reply README.md]: I missed this one ;)

**Comment by daKmoR** [comment]
we just merged a huge change to our documentation (moved from mdx -> md(js) 💪)

this MR is sadly effected and will need to be rebase and merge conflicts need to be resolved 😅
We are sorry for the inconvenience - if you need any help please let us know 🤗

**Comment by MathieuPuech** [comment]
> we just merged a huge change to our documentation (moved from mdx -> md(js) 💪)

That's a nice change, I think the rebase is good

**Comment by jorenbroekema** [comment]
Thanks 💪

**Pull request merged**